resource-definitions/template-driver/servicemonitor/servicemonitor-workload-def.yaml

```diff
@@ -0,0 +1,41 @@
+# This Resource Definition adds a Prometheus ServiceMonitor to a Workload
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: servicemonitor-workload
+entity:
+  driver_type: humanitec/template
+  name: servicemonitor-workload
+  type: workload
+  driver_inputs:
+    values:
+      namespace: ${resources['k8s-namespace#k8s-namespace'].outputs.namespace}
+      templates:
+        init: |
+          workload: {{ index (splitList "." "${context.res.id}") 1 }}

```

Comment:
Better to use ` {{ .resource.id }}` from the workload resource itself. https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#workload

Reply:
Good call!